### PR TITLE
Support perl 5.32 and Fedora

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds support for Fedora Perl-5.32 container image.

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
The tests are failing on `psgi` and `psgi_variables` for reason:

```bash
[Mon May 16 12:30:19.267839 2022] [lbmethod_heartbeat:notice] [pid 1:tid 1] AH02282: No slotmem from mod_heartmonitor
[Mon May 16 12:30:19.292391 2022] [mpm_event:notice] [pid 1:tid 1] AH00489: Apache/2.4.53 (Fedora) mod_fcgid/2.3.9 configured -- resuming normal operations
[Mon May 16 12:30:19.293396 2022] [core:notice] [pid 1:tid 1] AH00094: Command line: 'httpd -C Include /opt/app-root/etc/httpd.conf -D FOREGROUND'
/usr/bin/env plackup -s FCGI ./application.psgi

/usr/bin/env: 'plackup': No such file or directory
[Mon May 16 12:30:26.791803 2022] [fcgid:warn] [pid 19:tid 271] (104)Connection reset by peer: [client 172.17.0.1:38966] mod_fcgid: error reading data from FastCGI server
```

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
[test]

File: 5.32/Dockerfile.fedora [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+# This image provides a Perl 5.32 environment you can use to run your Perl applications.
+EXPOSE 8080
+
+# Image metadata
+ENV PERL_VERSION=5.32 \
+    PERL_SHORT_VER=532 \
+    VERSION=0 \
+    NAME=perl \
+    PATH=$PATH:/opt/app-root/src/extlib/bin \
```
Comment (michal-josef-spacek) [inline 5.32/Dockerfile.fedora:11]: @phracek Rewrite to "PATH=/opt/app-root/src/extlib/bin:$PATH \"

Comment (michal-josef-spacek) [reply 5.32/Dockerfile.fedora:11]: This is the root cause of failure

**Comment by phracek** [comment]
[test-all]

**Comment by michal-josef-spacek** [comment]
[test]

**Comment by phracek** [comment]
RHEL8 failures are fixed by #240

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
[test-all]

**Pull request merged**